light-client/eth-light-client/src/server.rs

```diff
@@ -0,0 +1,74 @@
+use ethers::{abi::AbiEncode, types::H256};
+use eyre::Result;
+use salvo::prelude::*;
+use serde::Deserialize;
+use serde_json::json;
+
+use crate::{config::Config, db::DB, merkle};
+
+#[derive(Deserialize)]
+struct VerifyReq {
+	indices: Vec<u32>,
+	hashes: Vec<H256>,
+}
+
+#[handler]
+async fn status() -> &'static str {
+	"ok"
+}
+
+#[handler]
+async fn root(dep: &mut Depot) -> String {
+	let db = dep.obtain::<DB>().expect("get DB");
+	let logs = db.select_logs().expect("get logs");
```

Comment:
is it safe to use expect with this server?

Reply:
I changed it to an HTTP response with the status 404 and an error message.